tests/data/example.model.jsonld

```diff
@@ -72,6 +74,7 @@
                 "@id": "http://schema.biothings.io"
             },
             "sms:displayName": "Patient ID",
```

Comment:
Why are you adding the primary key to the standard data model? The CSV will not match. Why not make another `database_model.model.jsonld` that matches that csv??

Reply:
I didn't add this (manually) one fo the tests must have?